### PR TITLE
Control if a listing can be exported in integrations

### PR DESCRIPTION
**Pull request by joaodubas**
This PR enable the control of `isExportable` flag from listing, allowing anyone to set if a list can be exported to other sites.

### CODE REVIEW

File: components/listings/EditProperties/index.js [hunk 1]
```diff
@@ -132,6 +134,30 @@ export default class EditListingProperties extends PureComponent {
               )}
             />
           </Col>
+          <Col width={[1, null, 1 / 3]}>
+            <Field
+              name="isExportable"
+              render={() => (
+                <FieldContainer label="Exportável">
```
Comment (joaodubas) [inline components/listings/EditProperties/index.js]: I really don't think this name is good, suggestions are accepted.

File: components/listings/Properties/index.js [hunk 2]
```diff
@@ -57,9 +59,16 @@ export default function ListingProperties({listing}) {
   return (
     <Fieldset>
       <Row flexDirection={['column', null, 'row']}>
-        <Col width={[1, null, 2 / 3]} mr={[0, null, 3]}>
+        <Col width={[1, null, 1 / 3]} mr={[0, null, 3]}>
           <Property name="Tipo de imóvel" value={listing.type} />
         </Col>
+        <Col width={[1, null, 1 / 3]}>
+          <Property
+            name="Exportável"
```
Comment (joaodubas) [inline components/listings/Properties/index.js]: The same here, this name isn't good.

File: components/listings/EditProperties/index.js (hunk 1 above)
Comment (gabriel-laet) [reply components/listings/EditProperties/index.js]: LGTM

File: components/listings/Properties/index.js (hunk 2 above)
Comment (gabriel-laet) [reply components/listings/Properties/index.js]: Maybe we can change the label to `Publicar em portais` ou `Ativar integração`

Comment (joaodubas) [reply components/listings/Properties/index.js]: Nice one, will use
`Integração ativada` here and `Ativar integração` when editing.

**Pull request merged**